CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Solid React Components Library
 
+## 0.2.4 ( February 27, 2019 )
```

Comment:
I'm going to change this to go into the RC so please remove this for now, as it will be in 0.3.0 and we can re-add this during the next release.

Reply:
Removed it.